CHANGELOG.md

```diff
@@ -432,4 +432,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [1.0.3]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.0.2...1.0.3
 [1.0.2]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.0.1...1.0.2
 [1.0.1]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.0.0...1.0.1
-[1.0.0]: https://github.com/rokwire/rokwire-building-blocks-api/releases/tag/1.0.0
+[1.0.0]: https://github.com/rokwire/rokwire-building-blocks-api/releases/tag/1.0.0
```

Comment:
I think this particular change is not needed. Please update the CHANGELOG with an entry for this task. Thanks.